x/tunnel/keeper/helper.go

```diff
@@ -72,3 +73,15 @@ func calculateDeviationBPS(oldPrice, newPrice sdkmath.Int) sdkmath.Int {
 
 	return newPrice.Sub(oldPrice).Abs().MulRaw(10000).Quo(oldPrice)
 }
+
+// IsOutOfGasError checks if the error object is an out of gas or gas overflow error type
+func IsOutOfGasError(err any) (bool, string) {
```

Comment:
why use any not use error?

Reply:
It checks against object received from `recover()` and the SDK errors here are not actually errors. (They don't implement error interface) [ref](https://github.com/bandprotocol/private-chain/blob/ba804b4d36e27bd8b4bc213c144c9618c44133cf/pkg/ctxcache/ctx_cache.go#L51)